plugin/path_role.go

```diff
@@ -480,8 +502,8 @@ func (b *GcpAuthBackend) pathRoleEditGceLabels(ctx context.Context, req *logical
 	return b.storeRole(ctx, req.Storage, roleName, role, warnings)
 }
 
-// role reads a gcpRole from storage. This assumes the caller has already obtained the role lock.
-func (b *GcpAuthBackend) role(ctx context.Context, s logical.Storage, name string) (*gcpRole, error) {
+// retrieveRole from storage. This assumes the caller has already obtained the role lock.
+func (b *GcpAuthBackend) retrieveRole(ctx context.Context, s logical.Storage, name string) (*gcpRole, error) {
```

Comment:
Can you share the rational for renaming this method?

Reply:
Generally I ascribe to the idea that function names should be read like verbs.

Reply:
I don't necessarily disagree, however this breaks a common naming convention in most of our secret backends. I think `Role` is pretty Go idiomatic as well: 

- https://golang.org/doc/effective_go.html#Getters

Reply:
I agree with the naming convention for getters, however I don't agree that this is a getter as described. This is retrieving the role from the storage backend, not returning an existing, unexported field on the `GcpAuthBackend`.

Reply:
That's fair, it's not a true getter as described in that link. I do feel that `role` is consistent with most of our other secret backends as well as some other methods/functions in this plugin (`Factory()`, `Backend()`, `IAMClient()`,`ComputeClient()`)

Reply:
Changed back to `role` for consistency across plugins that I wasn't aware of previously.